packages/ember-testing/lib/helpers.js

```diff
@@ -207,8 +207,10 @@ function wait(app, value) {
 
     // Every 10ms, poll for the async thing to have finished
     var watcher = setInterval(function() {
+      var router = app.__container__.lookup('router:main');
+
       // 1. If the router is loading, keep polling
-      var routerIsLoading = !!app.__container__.lookup('router:main').router.activeTransition;
+      var routerIsLoading = router.router && !!router.router.activeTransition;
```

Comment:
at the very least post initializers running

(that being said this is still better then the existing, so i am fine with merging without fixing ^^)